lib/devices/MiioVacuum.js

```diff
@@ -309,18 +365,36 @@ class MiioVacuum {
     }
 
     /**
+     * @typedef AppLocale
+     * @property {string} name
+     * @property {string} bom
+     * @property {string} location
+     * @property {string} language
+     * @property {string} timezone
+     * @property {string} logserver
+     */
+    /**
+     * @public
      * @abstract
-     * @returns {Promise<object>}
+     * @returns {Promise<AppLocale>}
      */
     async getAppLocale() {
         throw new NotImplementedError();
     }
 
+    /**
+     * @typedef Timer
+     * @property {string} id
+     * @property {string} cron
```

Comment:
I assume that’s the timer-spec in crontab format?

Reply:
Yes